docker-compose.yml.mustache

```diff
@@ -2,16 +2,11 @@
 
 couchdbprimary:
   build: roles/couchdb
+  volumes:
+    - /usr/local/lib/npmo/couchdb:/usr/local/var/lib/couchdb
   ports:
     - "55984:5984"
 
-couchdbsecondary:
```

Comment:
This breaks the replication tooling, but it wasn't ideal anyway. However, for the Twitter deployment, I'm going to have 4 daisy-chained Couch instances so we need a strategy for handling that use case. It might end up being some tooling around this repo.